corehq/apps/ota/views.py

```diff
@@ -238,7 +238,7 @@ def get_restore_response(domain, couch_user, app_id=None, since=None, version='1
     return restore_config.get_response(), restore_config.timing_context
 
 
-@login_or_digest_or_basic_or_apikey()
+@login_or_digest_or_basic_or_apikey
```

Comment:
i think these all need to support mobile (though it looks like ones i left out of my initial pr)